packaging/language/maven_artifact.py

```diff
@@ -26,6 +26,8 @@
 import hashlib
 import sys
 import posixpath
+from ansible.module_utils.basic import *
+from ansible.module_utils.urls import *
```

Comment:
that reason is gone, since 2.1 and ziploader this is now an actual import and not a 'fake import that is really a copy+paste'. The reason was for better debugging as it affected line number reporting.